### ISSUE TITLE
problems about base_lr set

### ISSUE THREAD

**Issue by ujsyehao** [weiliu89/caffe#865]
The author provides ssd_pascal.py to train model with finetuning VGG classification network, 
so it sets base_lr to 0.001(we set base_lr to 0.01 often if we train from scratch), 
but In add extra layers, lr_mult is set to 1, 
Shouldn't it be set to 10? 
because these new layers train from scratch and need higher learning rate.

**Comment by ujsyehao** [comment]
Does anyone help me?
